src/fides/api/alembic/migrations/versions/d0cbfec0b2dd_add_okta_to_connection_types.py

```diff
@@ -0,0 +1,62 @@
+"""add okta to connection_types
+
+Revision ID: d0cbfec0b2dd
+Revises: c9c72b3d550b
+Create Date: 2025-04-23 15:22:57.920715
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "d0cbfec0b2dd"
+down_revision = "c9c72b3d550b"
+branch_labels = None
+depends_on = None
+
+type_to_handle = "okta"
+
+
+def upgrade():
+    # add 'okta' to ConnectionType enum
+    conn = op.get_bind()
+    result = conn.execute("SELECT enum_range(NULL::connectiontype)").scalar()
+    current_values = result[1:-1].split(",")  # Remove the {} and split by comma
```

Comment:
potential edge case (maybe less for the current migration, but if we wanted to reuse this for future migrations, as it seems is your intention): can a particular enum value have a comma in it? if so, i think it'd break this, right?